cmake/std/atdm/sems-rhel6/environment.sh

```diff
@@ -214,7 +215,6 @@ export ATDM_CONFIG_MPI_PRE_FLAGS="--bind-to;none"
 
 export ATDM_CONFIG_WORKSPACE_BASE_DEFAULT=/home/atdm-devops-admin/jenkins
 export ATDM_CONFIG_TRIL_CMAKE_INSTALL_PREFIX_DATE_BASE_DEFAULT=/home/atdm-devops-admin/trilinos_installs
```

Comment:
This could be removed.

Reply:
Actually, that is pointing to `/home/atdm-devops-admin/` instead of the default location of `/projects/atdm_devops/`.